.gitignore

```diff
@@ -3,4 +3,6 @@ bin/
 /hack/operator-secret.yaml
 /charts/btp-manager/
 .DS_Store
-.idea/
\ No newline at end of file
+mod/
+template.yaml
```

Comment:
this will exclude `template.yaml` in every directory, do we want to restrict to certain path? you can do that by specifying full relative path to root dir.

Reply:
you are right